backend/app_tests/api/test_api_evidences.py

```
@@ -160,6 +160,52 @@ def test_create_evidences(self, test):
     def test_update_evidences(self, test):
         """test to update evidences with the API with authentication"""
 
+        folder = Folder.objects.create(name="test2")
```

Ruff (0.8.2)

Local variable `folder` is assigned to but never used

Remove assignment to unused variable `folder`

(F841)